components/dialog/header.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Header
+page_title: Dialog Header
+description: Header of the Dialog for Blazor.
+slug: dialog-header
+tags: telerik,blazor,dialog,header
+published: True
+position: 5
+---
+
+# Dialog Header
+
+The header contains the `Title` and the `Close Action` button.
+
+The Dialog supports the title configuration through a template and a string parameter. The default value is `null(default(string))`.
+
+You can control the close action via the `ShowCloseButton` parameter. Its default value is `true`.
+
+>important If you don't want to render the header, set the `ShowCloseButton` to `false` and leave the title empty.
```

Comment:
This should look like a tip, not a warning. Also, does it make sense to highlight it? This doesn't look like a common use case.

```suggestion
> If you don't want to render the header, set the `ShowCloseButton` to `false` and leave the title empty.
```